app/components/ui/BackgroundRays/styles.module.scss

```diff
@@ -0,0 +1,251 @@
+.rayContainer {
+  --gradient-opacity: 0.8;
+  --primary-color: rgba(147, 112, 219, var(--gradient-opacity));
+  --secondary-color: rgba(138, 43, 226, var(--gradient-opacity));
+  --accent-color: rgba(180, 170, 220, var(--gradient-opacity));
+
+  // Theme-specific colors
+  --ray-color-primary: color-mix(in srgb, var(--primary-color), transparent 30%);
+  --ray-color-secondary: color-mix(in srgb, var(--secondary-color), transparent 30%);
+  --ray-color-accent: color-mix(in srgb, var(--accent-color), transparent 30%);
+
+  // Theme-specific gradients
+  --ray-gradient-primary: radial-gradient(var(--ray-color-primary) 0%, transparent 70%);
+  --ray-gradient-secondary: radial-gradient(var(--ray-color-secondary) 0%, transparent 70%);
+  --ray-gradient-accent: radial-gradient(var(--ray-color-accent) 0%, transparent 70%);
+
+  position: fixed;
+  inset: 0;
+  overflow: hidden;
+  animation: fadeIn 1.5s ease-out;
```

Comment:
[flash.webm](https://github.com/user-attachments/assets/4605ce67-fd4c-44be-9b13-eded0cc094ae)


![FlashBANG](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExZG9td2Jla3A3dmt4dGRuYzVwN3BheHVxNG9vYmp3N3EyaTdycGNiZCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/irOE2ho4vMdEqsCdly/giphy.gif)



This white flash happening on page reload is in my opinion too aggressive.

> A minimal, distraction-free UX design is essential because it allows users to focus entirely on their goals without being overwhelmed by unnecessary elements. By removing clutter and prioritizing simplicity, such designs reduce cognitive load, enhance usability, and create a seamless, stress-free interaction. This approach not only improves efficiency but also fosters a more enjoyable and meaningful user experience, where functionality and purpose take precedence.

```suggestion
```

Reply:
yeah I also dont like the flash.. but its therre from the start and didn't want to change the existing stuff only addition of the background.

Reply:
I don't think it was there before, the line
`animation: fadeIn 1.5s ease-out;` in the  `.rayContainer` is responsible for this white flash as it transitions from default white background to black on page reload.

Reply:
Okayy.. so the chat component had some background that was blocking the effects background. and after removing that the only thing that was providing the color was the effects background. when the opacity was animated from 0 to 1 it basically started from transparent exposing the white background from the parent component. fixed it by removing the background color from the effect's container to the parent container 

thanks for the feedback